### DISCUSSION TITLE
Welcome to micropython-nano-gui Discussions!

### DISCUSSION THREAD

**Discussion by peterhinch** [peterhinch/micropython-nano-gui#4] [General]
Just testing this new feature. Comments welcome

**Comment by askpatrickw** [comment]
Very curious to see how it goes! I like that the discussions will show up alongside the other Github notifications.

**Comment by peterhinch** [comment]
I suspect that the forum is the best place for open discussions as it has the widest readership, so I've only enabled it in this repo. I'll see how it goes.
